### DISCUSSION TITLE
用窮舉法解決版權問題的神奇專案

### DISCUSSION THREAD

**Discussion by MAKERRRofKEVIN** [1101-GS4719-C/Python-Community-GS4719-C#77] [科技新知]
9月剛寫的文章，關於音樂版權與程式，順便推一下我們的自媒體。
---
All the music lcc 是一個由兩位英國青年合作的專案，他們透過自身的法律及資工專業試圖修正存在已久的著作權問題，他們所採用的路徑可說是最簡單也是最複雜的-將所有的旋律窮舉出來。
https://www.instagram.com/p/CUsJH_GPNQ8/?utm_source=ig_web_copy_link

**Comment by madeuf** [comment]
感覺很酷欸，我小時候寫過的歌會不會也在裡面啊

**Comment by HSU-YU-JEN** [comment]
想出這個方法的人真的好強 窮舉六百億次!!

**Comment by swerfg03joe** [comment]
音樂這種東西要看類型，流行歌曲就短短幾分鐘，若是使用吉他曲子調性一決定，後面就是那幾個和弦的排列組合了，要怎麼界定侵權是一個很難的問題
